### DISCUSSION TITLE
[Help] Discord Invite is Invalid

### DISCUSSION THREAD

**Discussion by mccullohb** [fomantic/Fomantic-UI#1553] [General] [answered]
# Help Wanted

Both invite links to the Discord Server (Homepage and Issues Page) are invalid.

## Testcase
https://discord.com/invite/YChxjJ3

**Comment by lubber-de** [comment]
I unfortunately cannot reproduce this, both links are still valid for me 🤔 
Maybe there was some discord issue when you tried.
Could you please try again? If it is still not working, please tell us your discord username, maybe we can figure something out
Please also post a screenshot of the error message in case it still does not work for you. Thanks

**Comment by mccullohb** [comment, marked as answer]
> I unfortunately cannot reproduce this, both links are still valid for me 🤔
> Maybe there was some discord issue when you tried.
> Could you please try again? If it is still not working, please tell us your discord username, maybe we can figure something out
> Please also post a screenshot of the error message in case it still does not work for you. Thanks

Hey! I tried again and I was able to join :) Not quite sure what happened but it's working now. I'll go ahead and close this.
